.github/workflows/pages.yml

```
@@ -26,10 +26,31 @@
+          export PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1
+          export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1
+          export NODE_ENV=production
+          # Create .npmrc with aggressive hoisting for native bindings
+          echo "shamefully-hoist=true" > .npmrc
+          echo "auto-install-peers=true" >> .npmrc
+          echo "prefer-frozen-lockfile=false" >> .npmrc
+          # Install with multiple fallback strategies
+          pnpm install --no-frozen-lockfile --ignore-scripts || \
+          (rm -rf node_modules && pnpm install --no-frozen-lockfile) || \
+          (rm -rf node_modules pnpm-lock.yaml && pnpm install)
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)